Code/IO/include/sitkImageFileReader.h

```diff
@@ -40,7 +40,10 @@ namespace itk {
      *
      * An interface is also provided to access the information from
      * the underlying itk::ImageIO. This information can be loaded
-     * with the ReadImageInformation method.
```

Comment:
"The information is from the from" -> "The information is from the itk::ImageIO..."

"In some degenerate cases the reading the bulk data may produce difference results." -> 
"In some degenerate cases reading the bulk data may produce different results.
"